addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18153,3 +18153,23 @@ msgstr ""
 msgctxt "#38025"
 msgid "Choose information provider"
 msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38026"
+msgid "Decode the stereo stream from 3D files"
```

Comment:
well, 3D Blu-rays are not the standard Blu-ray format and at least over hear at least 10€ more expensive. So I really doubt that people not interested in 3D will buy/archive 3D Blu-Rays. Please don't get me wrong, the setting itself is not a bad thing, I just have the feeling that it's only ever needed by maybe 1% of users that collect 3D material but watch it in 2D only.